### ISSUE TITLE
generate PNN (based on 1) 에서 convention을 비워두는 문제: genPNN에서 변환표 선택 가능하도록 #2와 연관

### ISSUE THREAD

**Issue by sunhoi** [stannam/KPNN#13]
convention = "", 이렇게 하고 다른 발음기호 선택했더니, 
Error in txtProgressBar(max = length(x), style = 3) : 
  must have 'max' > 'min'
Called from: txtProgressBar(max = length(x), style = 3)
Browse[1]> 
와 같은 표시가 나오네.

**Comment by sunhoi** [comment]
이 문제는 해결된건가? 해결되었다면 무엇이 문제였고, 어떻게 수정된 것인지?

**Comment by stannam** [comment]
제 컴퓨터에서는 되는 것 같은데 확인 부탁드립니다^^

**Comment by sunhoi** [comment]
비워 두었을 경우는 해결된 거 같은데, 다른 것을 채웠을 때, 예를 들면, IPA를 채웠을 때코드가 돌아가는 도중 한글자모-ipa 대응표를 중간에 선택하지 않고 바로 klat을 입력한 결과가 나오는데, 왜 그런 현상이 발생하는지?  즉, 해당 파일을 선택하게 하는 과정 없이 그냥 klat을 입력한 결과가 나옴.

**Comment by sunhoi** [comment]
hangul_converter.r 코드에서 

transcription_location <<- transcription_location 이렇게 되어 있는데, transcription_location <- transcription_location 되어야 하는 건 아닌지, 아니면 그냥 두어도 괜찮은지?

**Comment by stannam** [comment]
> 비워 두었을 경우는 해결된 거 같은데, 다른 것을 채웠을 때, 예를 들면, IPA를 채웠을 때코드가 돌아가는 도중 한글자모-ipa 대응표를 중간에 선택하지 않고 바로 klat을 입력한 결과가 나오는데, 왜 그런 현상이 발생하는지? 즉, 해당 파일을 선택하게 하는 과정 없이 그냥 klat을 입력한 결과가 나옴.

제 컴퓨터에서는 그런 문제가 생기지 않네요. 내일 한번 같이 봐야할 것 같습니다.
매뉴얼 genPNN 항목의 3번예시와 같이 parameter 설정하시면 될 것 같습니다. 아래에 굵은 글씨로 되어있는 부분을 참고해주세요.
net <- genPNN(data = corpus, 
              entry = "entries", 
              **convention = "IPA",** 
              **unit = "IPA",** 
              deletion = T,
              pajek = T)

**Comment by stannam** [comment]
> hangul_converter.r 코드에서
> 
> transcription_location <<- transcription_location 이렇게 되어 있는데, transcription_location <- transcription_location 되어야 하는 건 아닌지, 아니면 그냥 두어도 괜찮은지?

이부분은 그대로 두어야 합니다..

**Comment by sunhoi** [comment]
제 컴퓨터에서는 그런 문제가 생기지 않네요. 내일 한번 같이 봐야할 것 같습니다.
매뉴얼 genPNN 항목의 3번예시와 같이 parameter 설정하시면 될 것 같습니다. 아래에 굵은 글씨로 되어있는 부분을 참고해주세요.
net <- genPNN(data = corpus,
entry = "entries",
convention = "IPA",
unit = "IPA",
deletion = T,
pajek = T)

이렇게 했는데도 비워 두거나, Klat을 넣은 것과 동일한 결과가 나옴. 내일 살펴보자.

**Comment by sunhoi** [comment]
r-studio를 처음 작동할 때는 작동함. 그러나 그 뒤부터는 작동하지 않음.

**Comment by stannam** [comment]
무엇이 문제인지 파악한 것 같습니다. 처음 돌릴때 대응표 파일을 지정해주면 이후에도 그게 계속 남아서 그렇습니다.

**Comment by stannam** [comment]
이제 아마 제대로 작동할 것입니다.

**Comment by sunhoi** [comment]
이제는 Klat을 넣어도 어떤 파일을 선택할지를 물어 보네.

**Comment by stannam** [comment]
네.. 대문자 Klat을 넣었을 때에는 변환표를 선택하는 것이 맞습니다. "convention = klat" convention에 값을 지정하지 않을 때에는 내장된 klattese변환표를 불러오고 나머지 경우에는 모두 파일 선택을 하는 것으로 디자인했습니다.
